### PR TITLE
test: Tests for swirlds.log on Turtle

### LINKED ISSUE #21336: Tests for Turtle's swirlds.log
**Requirements for `swirlds.log`** 

- Messages without markers should not be in this log
- Messages with enabled markers should be in this log
- Only messages with the node ID in the thread context should appear
- Info level and more critical

**Tests**

- Run happy path test (as a normal unit test), read the log from disk
- ensure a specific set of markers are present
- ensure each message is from the correct node
- ensure correct levels
- check presence of messages that are always logged

### PR DESCRIPTION
**Pull request by netopyr**
**Description**:

This PR adds tests for `swirlds.log` on Turtle.

**Related issue(s)**:

Fixes #21336

### CODE REVIEW

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+            List.of(STARTUP, PLATFORM_STATUS, STATE_TO_DISK, MERKLE_DB);
+
+    /**
+     * Sanity test with a single node to verify basic logging functionality.
+     *
+     * <p>This test verifies that a single node can log messages with allowed markers
+     * and that these messages appear in the correct swirlds.log file.
+     */
+    @Test
+    void testSingleNodeLogsAllowedMarkersToSwirdsLog() throws Exception {
```
Comment (poulok) [inline platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java:53]: I don't think we need a separate test for a single node network.

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    }
+
+    /**
+     * Test that only INFO level and above messages are logged to swirlds.log.
+     *
+     * <p>This test verifies that the log level threshold is correctly set to INFO.
+     */
+    @Test
+    void testLogLevelThresholdIsInfo() throws Exception {
+        final TestEnvironment env = new TurtleTestEnvironment(0L);
```
Comment (poulok) [inline platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java]: If all our tests that require an environment to be created and started start taking too long, we can easily combine multiple checks into a single test.

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+    }
+
+    /**
+     * Test the folder structure of build/turtle to verify it contains only node directories.
+     *
+     * <p>This test verifies that the build/turtle directory structure is correct
+     * and doesn't contain unexpected directories like node-unknown.
+     */
+    @Test
+    void testBuildTurtleFolderStructure() throws Exception {
```
Comment (poulok) [inline platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java]: It seems excessive to have separate tests for all these little checks. We should just add this check to one of the previous tests.

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java (hunk 1 above)
Comment (netopyr) [reply platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java:53]: I think it is always good practice to have tests for edge cases.

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java (hunk 2 above)
Comment (netopyr) [reply platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java]: I hope we will never reach a state where this is something we have to consider. 😄

File: platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java (hunk 3 above)
Comment (netopyr) [reply platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/fixtures/turtle/logging/TurtleSwirdsLogTest.java]: I merged the tests, but, honestly, I am not very happy with the outcome. Hopefully, I never have to look at these tests again.

**Review by andrewb1269hg — APPROVED** [review]
Approved `platform-sdk/consensus-otter-tests/build.gradle.kts`
